thunder/executors/transformer_engineex.py

```diff
@@ -219,6 +219,8 @@ def forward(self, inp, weight, bias, is_grad_enabled: bool = False):
                 grad_input_quantizer,
             ) = self._get_quantizers(is_grad_enabled)
 
+            input_quantizer.internal = True
```

Comment:
It would be better to update this line

https://github.com/Lightning-AI/lightning-thunder/blob/1ef2b946f25400ed32df83674c41f9ad0b3250ad/thunder/executors/transformer_engineex.py#L293

Reply:
oh, yes of course.